src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -3,7 +3,7 @@ SETLOCAL
 
 SET PACKAGE_DIR=%1
 SET EXECUTION_DIR=%2
-
+set PACKAGE_DIR=%PACKAGE_DIR:/=\%
```

Comment:
* Normalizes all slashes to \ for Windows, preventing weird issues with the paths passed in.